src/containers/BooksList.js

```diff
@@ -1,16 +1,21 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { removeBookAction } from '../actions/index';
+import { removeBookAction, changeFilterAction } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
 /* eslint-disable no-unused-vars */
```

Comment:
- [ ] Why are you disabling this rule? I can't see any variables you are not using here :thinking:

Reply:
Previously, I was using the wrong eslint file, that's why they gave an error, now it's fine.

Thank you